hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityNullTest_2_0_1.java

```diff
@@ -54,14 +54,15 @@ public class ClientCompatibilityNullTest_2_0_1 {
     @Before
     public void setUp() throws IOException {
         File file = new File(getClass().getResource("/2.0.1.protocol.compatibility.null.binary").getFile());
-        InputStream inputStream = new FileInputStream(file);
-        byte[] data = new byte[(int) file.length()];
-        inputStream.read(data);
-        ByteBuffer buffer = ByteBuffer.wrap(data);
-        ClientMessageReader reader = new ClientMessageReader(0);
-        while (reader.readFrom(buffer, true)) {
-            clientMessages.add(reader.getClientMessage());
-            reader.reset();
+        try (InputStream inputStream = new FileInputStream(file)) {
```

Comment:
```suggestion
        try (InputStream inputStream = getClass().getResourceAsStream("/2.0.1.protocol.compatibility.null.binary")) {
```

This avoids a `FileInputStream` entirely.

Same comment applies to all `ClientCompatibilityNullTest`/`MemberCompatabilityTest`s

Reply:
rev : 800db57b27c01d717ef7c7ee26b0f9497e1f0e88